lang/test/org/partiql/lang/eval/evaluatortestframework/EvaluatorTestCase.kt

```diff
@@ -59,6 +59,9 @@ data class EvaluatorTestCase(
      * `false`.  Note that, when `false`, [expectedPermissiveModeResult] is ignored.
      */
     override val implicitPermissiveModeTest: Boolean = true,
+
+    override val target: EvaluatorTestTarget = EvaluatorTestTarget.ALL_PIPELINES,
```

Comment:
Could add kdoc for `target`. `EvaluatorErrorTestCase` includes kdoc for `target`. Also nit: `target` could be renamed `targetPipeline` for further specificity

Reply:
done.